src/components/common/gettingStartedCard/GettingStarted.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import GettingToast from '../../../assets/img/lifebuoy.png'
+import { updateLoginCount } from '../../../services/service'
+import { setActionWithExpiry } from '../helpers/Helpers'
+import './gettingStarted.scss'
+
+export interface GettingStartedType {
+    className: string
+    showHelpCard: boolean
+    hideGettingStartedCard: () => void
+    loginCount: number
+}
+
+function GettingStartedCard({ className, hideGettingStartedCard }: GettingStartedType) {
+    const onClickedOkay = () => {
+        setActionWithExpiry('clickedOkay', 1)
+        hideGettingStartedCard()
+    }
+
+    const onClickedDontShowAgain = () => {
+        let updatedPayload = {
+            key: 'login-count',
+            value: '5',
+        }
+        updateLoginCount(updatedPayload)
+        hideGettingStartedCard()
+    }
+
+    return (
+        <div className={`${className} getting_tippy__position getting-started-card cn-0 p-20 br-8 fs-13 `}>
```

Comment:
Use the `className` prop at the end of other class names. It'll help override any property using classes present in the prior classes.